Formula/h2spec.rb

```diff
@@ -0,0 +1,29 @@
+class H2spec < Formula
+  desc "Fast website link checker in Go"
+  homepage "https://github.com/summerwind/h2spec"
+  url "https://github.com/summerwind/h2spec.git",
+    tag:      "v2.6.0",
+    revision: "70ac2294010887f48b18e2d64f5cccd48421fad1"
+  license "MIT"
+  head "https://github.com/summerwind/h2spec.git"
+
+  depends_on "go" => :build
+
+  def install
+    commit = Utils.safe_popen_read("git", "rev-parse", "--short", "HEAD").chomp
```

Comment:
I might see if I could add a `git_commit` helper function or something, but that's not blocking for this PR

Reply:
second to this idea.

Also might be good to include helper function for shell completions as well